themes/bootstrap3/scss/bootstrap.scss

```diff
@@ -119,7 +119,7 @@ label { cursor: pointer; }
 .navbar-nav button.dropdown-toggle {
   @include button-variant($btn-default-color, $btn-default-bg, $btn-default-border);
   appearance: none;
-  padding: 15px;
+  padding: $navbar-padding-vertical;
```

Comment:
this may need to be paired with a line-height adjustment to keep the text centered (as Susan noted).

Reply:
Oddly enough, I had to try really hard to reproduce this, but I finally managed. Perhaps Chrome on Windows is the only browser having this problem, but I'll see what needs to be done.

Reply:
Ok, got it. Needed to increase base input height.